src/components/pages/donation_form/DonorSummary.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+	<div class="donor-summary" v-if="hasData">
+		<h3>{{ $t('donation_summary_contact_data_header') }}</h3>
+		<p v-if="address.fullName">{{ address.fullName }}</p>
+		<p v-if="address.streetAddress">{{ cleanedStreetAddress }}</p>
+		<p v-if="address.postalCode || address.city">
+			{{ address.postalCode }} {{ address.city }}
+		</p>
+		<p v-if="country">{{ country }}</p>
+		<p v-if="email">{{ email }}</p>
+	</div>
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue';
+import { useI18n } from 'vue-i18n';
+import { clearStreetAndBuildingNumberSeparator } from '@src/util/street_and_building_number_tools';
+import { Country } from '@src/view_models/Country';
+import type { Salutation } from '@src/view_models/Salutation';
+
+const props = defineProps<{
+	address: Record<string, string>;
+	countries: Array<Country>;
+	salutations: Array<Salutation>;
+}>();
+
+const { t } = useI18n();
+
+const cleanedStreetAddress = computed( () =>
+	clearStreetAndBuildingNumberSeparator( props.address.streetAddress || '' )
+);
+
+const email = computed(() =>
+	props.address.email || t('donation_confirmation_review_email_missing')
```

Comment:
This will show an error when someone fills out the address, not can't see that requirement in the ticket